modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ViewObjectDefinitions.tsx

```diff
@@ -327,21 +323,19 @@ export default function ViewObjectDefinitions({
 	};
 
 	useEffect(() => {
-		if (Liferay.FeatureFlags['LPS-148856']) {
-			const makeFetch = async () => {
-				API.getAllObjectFolders().then((response) => {
-					setObjectFoldersRequestInfo(response);
-					setSelectedObjectFolder(response.items[0]);
-					setLoading(false);
-				});
+		const makeFetch = async () => {
+			API.getAllObjectFolders().then((response) => {
+				setObjectFoldersRequestInfo(response);
+				setSelectedObjectFolder(response.items[0]);
+				setLoading(false);
+			});
 
-				const objectDefinitions = await API.getAllObjectDefinitions();
+			const objectDefinitions = await API.getAllObjectDefinitions();
 
-				setObjectDefinitionActions(objectDefinitions.actions);
-			};
+			setObjectDefinitionActions(objectDefinitions.actions);
+		};
```

Comment:
Since you are at it, I suggest you to move the `objectDefinitions` const declaration closer to its usage:

```javascript
const makeFetch = async () => {
	const allObjectFolders = await API.getAllObjectFolders();

	setObjectFoldersRequestInfo(allObjectFolders);
	setSelectedObjectFolder(allObjectFolders.items[0]);
	
	const objectDefinitions = await API.getAllObjectDefinitions();
	
	setObjectDefinitionActions(objectDefinitions.actions);

	setLoading(false);
};
```

Reply:
Thank you all. I will do it

Reply:
@p-albuquerque  from what I'm seeing in the code, this `loading` state is to wait until the props that go inside the `ObjectFoldersSideBar` component, and all these three states `objectDefinitionsActions`, `selectedObjectFolder` and `objectFoldersRequestInfo` will be used inside the `ObjectFoldersSideBar`, so we can follow @MarinhoFeliphe's suggestion of moving the const `objectDefinitions` declaration closer to its use, but the `setLoading` state should be last in this function to avoid synchronization problems.

```tsx
	const makeFetch = async () => {
		const allObjectFolders = await API.getAllObjectFolders();
	
		setObjectFoldersRequestInfo(allObjectFolders);
		setSelectedObjectFolder(allObjectFolders.items[0]);
		
		const objectDefinitions = await API.getAllObjectDefinitions();
		
		setObjectDefinitionActions(objectDefinitions.actions);

		setLoading(false);
	};
```